hack/tools/logcheck/pkg/logcheck.go

```diff
@@ -0,0 +1,489 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkg
```

Comment:
In some places `pass.Report.Message` starts with capital letter and ends with fullstop. Some don't.
We should keep it uniform.

Like
```
pass.Report(analysis.Diagnostic{
  Pos:     fun.Pos(),
  Message: fmt.Sprintf("function %q should not be used, convert to contextual logging", fName),
})
```
and
```
pass.Report(analysis.Diagnostic{
  Pos:     fun.Pos(),
  Message: fmt.Sprintf("Additional arguments to %s should always be Key Value pairs. Please check if there is any key or value missing.", funName),
})
```

Reply:
Good point. Looking at https://grep.app/search?q=pass.Report it seems that lowercase is more common.

Reply:
Actually, all of the reports with uppercase fall into that category. I added the missing full stop there.